frontend/src/component/insights/componentsChart/CreationArchiveChart/CreationArchiveChart.tsx

```diff
@@ -47,6 +50,8 @@ interface ICreationArchiveChartProps {
     isLoading?: boolean;
 }
 
+type DataResult = 'Not Enough Data' | 'Batched' | 'Weekly';
```

Comment:
I'd love a better name for this type if you can think of one 😅

Reply:
Local type, good enough I guess.